src/components/subscriptions/licenses/LicenseManagementModals/LicenseManagementRemindModal.jsx

```diff
@@ -31,27 +32,19 @@ const generateEmailTemplate = (contactEmail) => ({
 
 /**
  * Returns StatefulButton labels
- * @param {boolean} remindAllUsers
- * @param {number} userCount
  * @param {number} totalToRemind
  * @returns {Object}
  */
-const generateRemindModalSubmitLabel = (remindAllUsers, userCount, totalToRemind) => {
-  // If we are not reminding all users, use users.length
-  // Else if totalToRemind is passed use that
-  // Else use 'All'
-  let buttonNumberLabel = 'All';
-  if (!remindAllUsers) {
-    buttonNumberLabel = userCount;
-  } else if (totalToRemind > 0) {
-    buttonNumberLabel = totalToRemind;
+const generateRemindModalSubmitLabel = (totalToRemind) => {
+  let buttonNumberLabel = '(all)';
```

Comment:
Small nit, don't feel strongly about it all but this use to be `'(All)'` instead of `'(all)'` this is also the case in the revoke modal. There are no mocks in this state so I think both are fine.

Reply:
I believe for all our buttons only the first word should to be upper case. @adamstankiewicz

Reply:
@manny-m @long74100 I made the suggestion to change the casing to Long because typically our buttons only have the first letter capitalized. That said, when it does say "all", it should probably be without parentheses, e.g. "Revoke all" or "Remind all"